flocker/node/agents/functional/test_gce.py

```diff
@@ -144,6 +169,64 @@ def test_multiple_cluster(self):
                          list(x.dataset_id
                               for x in gce_block_device_api_2.list_volumes()))
 
+    def test_list_live_nodes_pagination_and_removal(self):
+        """
+        list_live_nodes should be able to walk pages to get all live nodes and
+        should not have nodes after they are destroyed.
+        """
```

Comment:
So spinning up a second node is expensive, and I need at least 2 live nodes in order to verify pagination is working... sort of bad, but I went for 1 large large test that tests everything that requires having a second node spun up rather than two proper unit tests.

I will ~~make a sacrifice to the testing gods in order to ask for forgiveness for this uncleanliness~~ add a comment reflecting this.